### PR TITLE
Fix issue 319

### PR DESCRIPTION
**Pull request by ruanitto**
## Pull Request (PR) Checklist
Please check if your pull request fulfills the following requirements:
- [x] The PR is submitted to the `main` branch.
- [x] I've read the [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md) and my PR doesn't break the rules.
- [x] I've read and understand the [Developer Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Guidelines.md).
- [x] I confirm that I've run the code locally and everything works as expected.
- [ ] 🎬 I've attached a **screen recoding** of the changes. 

> Tip: drag & drop the video to the PR description.

## What's changed?
 
Add translation for pt-BR

### CODE REVIEW

**Review by ILIYANGERMANOV — APPROVED** [review]

File: shared/ui/core/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+    <string name="export_csv_file_standard">Exportar arquivo CSV com opções padrão</string>
+    <string name="export_csv_file_standard_description">Use as opções padrão e certifique-se de incluir cabeçalhos.</string>
+    <string name="how_to_import">Como importar</string>
+    <string name="open">abrir</string>
+    <string name="steps">Passos</string>
+    <string name="how_to">Como fazer</string>
+    <string name="video">Vídeo</string>
+    <string name="article">Artigo</string>
+    <string name="upload_csv_file">Carregar arquivo CSV</string>
+    <string name="export_data">"Exportar Dados "</string>
```
Comment (lsfoschine) [inline shared/ui/core/src/main/res/values-pt-rBR/strings.xml:48]: <string name="export_data">"Exportar Dados"</string>

> Espaço extra removido.

File: shared/ui/core/src/main/res/values-pt-rBR/strings.xml [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+    <string name="category_name">Nome da categoria</string>
+    <string name="choose_category">Escolha a categoria</string>
+    <string name="description_text_field_hint">Insira quaisquer detalhes aqui</string>
+    <string name="clean_filter">Limpar filtro</string>
+    <string name="filter">Filtro</string>
+    <string name="apply_filter">Aplicar filtro</string>
+    <string name="by_type">Por tipo</string>
+    <string name="incomes">Rendas</string>
+    <string name="time_period">Período</string>
+    <string name="select_time_range">Selecione o intervalo de tempo</string>
```
Comment (lsfoschine) [inline shared/ui/core/src/main/res/values-pt-rBR/strings.xml:220]: <string name="select_time_range">Selecione o período</string>

File: shared/ui/core/src/main/res/values-pt-rBR/strings.xml [hunk 3]
```diff
@@ -0,0 +1,490 @@
...
+    <string name="authentication_required_description">Autenticar para acessar o aplicativo</string>
+    <string name="total_budget">Orçamento total</string>
+    <string name="category_budget">Orçamento da categoria</string>
+    <string name="multi_category_budget">Orçamento multicategoria (%1$s)</string>
+    <string name="borrowed_uppercase">EMPRESTADO</string>
+    <string name="lent_uppercase">LENT</string>
+    <string name="january">Janeiro</string>
+    <string name="february">Fevereiro</string>
+    <string name="march">Março</string>
+    <string name="april">abril</string>
```
Comment (lsfoschine) [inline shared/ui/core/src/main/res/values-pt-rBR/strings.xml:411]: <string name="april">Abril</string>

File: shared/ui/core/src/main/res/values-pt-rBR/strings.xml [hunk 4]
```diff
@@ -0,0 +1,490 @@
...
+    <string name="may">Maio</string>
+    <string name="june">Junho</string>
+    <string name="july">Julho</string>
+    <string name="august">agosto</string>
```
Comment (lsfoschine) [inline shared/ui/core/src/main/res/values-pt-rBR/strings.xml:415]: <string name="august">Agosto</string>

File: shared/ui/core/src/main/res/values-pt-rBR/strings.xml [hunk 5]
```diff
@@ -0,0 +1,490 @@
...
+    <string name="tomorrow_date">Amanhã, %1$s</string>
+    <string name="expired">Expirado</string>
+    <string name="authentication_succeeded">Autenticação bem-sucedida!</string>
+    <string name="authentication_failed">Falha na autenticação.</string>
+    <string name="notification_1">Você fez alguma transação hoje? 🏁</string>
+    <string name="notification_2">Você controlou suas despesas hoje? 💸</string>
+    <string name="notification_3">Você registrou suas transações hoje? 🏁</string>
+    <string name="cash">Dinheiro</string>
+    <string name="bank">Banco</string>
+    <string name="revoult">Revolução</string>
```
Comment (lsfoschine) [inline shared/ui/core/src/main/res/values-pt-rBR/strings.xml:352]: > No caso, não seria mais apropiado "Rotação", "Ciclo" ou "Giro"?

File: shared/ui/core/src/main/res/values-pt-rBR/strings.xml [hunk 6]
```diff
@@ -0,0 +1,490 @@
...
+    <string name="ivy_wallet_is_opensource_description">O código da Ivy Wallet é aberto e todos podem vê-lo. Acreditamos que transparência e ética são essenciais para qualquer produto de software. Se você gosta do nosso trabalho e deseja melhorar o aplicativo, você pode contribuir com nosso repositório público no Github.</string>
+    <string name="contribute">Contribuir</string>
+    <string name="adjust_balance">Ajustar saldo</string>
+    <string name="authentication_required">Autenticação necessária</string>
+    <string name="authentication_required_description">Autenticar para acessar o aplicativo</string>
+    <string name="total_budget">Orçamento total</string>
+    <string name="category_budget">Orçamento da categoria</string>
+    <string name="multi_category_budget">Orçamento multicategoria (%1$s)</string>
+    <string name="borrowed_uppercase">EMPRESTADO</string>
+    <string name="lent_uppercase">LENT</string>
```
Comment (lsfoschine) [inline shared/ui/core/src/main/res/values-pt-rBR/strings.xml:407]: <string name="lent_uppercase">EMPRESTEI</string>

File: shared/ui/core/src/main/res/values-pt-rBR/strings.xml [hunk 7]
```diff
@@ -0,0 +1,490 @@
...
+    <string name="add_tags">Adicionar tags</string>
+    <string name="this_transaction_is_associated_with_loan">* Esta transação está associada ao Empréstimo: %1$s</string>
+    <string name="this_transaction_is_associated_with_loan_record">* Esta transação está associada a um Registro de Empréstimo de Empréstimo: %1$s</string>
+    <string name="note_transaction_associated_with_loan_record_of_loan">Nota: Esta transação está associada a um Registro de Empréstimo de Empréstimo: %1$s\nVocê está tentando alterar a conta associada ao registro de empréstimo para uma conta de moeda diferente\n O Registro de Empréstimo irá ser recalculado com base nas taxas de câmbio atuais</string>
+    <string name="note_you_are_trying_to_change_the_account_associated_with_the_loan">Nota: Você está tentando alterar a conta associada ao empréstimo: %1$s com uma conta de moeda diferente, \nTodos os registros do empréstimo serão recalculados com base na moeda de hoje taxas de câmbio</string>
+    <string name="attributions">Atribuições</string>
+    <string name="icons">Ícones</string>
+    <string name="fonts">Fontes</string>
+    <string name="tech_stack">Pilha de tecnologia</string>
```
Comment (lsfoschine) [inline shared/ui/core/src/main/res/values-pt-rBR/strings.xml:463]: > Sugestão de tradução "Conjunto de tecnologias", "Pilha de tecnologias" ou "Ecossistema de tecnologias".

**Review by lsfoschine — COMMENTED** [review]
Aqui estão algumas ideias para melhorar a tradução e ajustes.

**Pull request merged**